data/homepage.yaml

```diff
@@ -12,7 +16,7 @@ bullhorn:
 platform:
   main: Get enterprise engineering and support from Red Hat with a platform that can tackle any automation challenge.
   discover: Ansible Automation Platform provides everything needed to create, execute, and manage automation in a single subscription. From execution environments to certified collections to automation analytics, discover the features and benefits of Ansible Automation Platform.
```

Comment:
discover: Red Hat Ansible Automation Platform ...

(need the full name on first mention).

Reply:
This isn't the first mention on the page though. The key below "learn:" is used in the banner at the very top of the page. Likewise there is the full branded name in the logo.

I feel like, in the context of the rendered site, this is fine. I also see that the value of this key has two instances of "AAP" but still think Red Hat gets plenty of mentions here.

Reply:
The first use is in the hero band at the top, that is the full text. Is that close enough to this paragraph to count, @cybette? What's the official short-name we should use elsewhere?